packages-ts/starknet-gauntlet-ledger/src/index.ts

```diff
@@ -13,6 +13,7 @@ import { Stark, LedgerError } from '@ledgerhq/hw-app-starknet'
 export const DEFAULT_LEDGER_PATH = "m/2645'/579218131'/894929996'/0'"
 export const LEDGER_PATH_REGEX = /^\s*m\s*\/\s*2645\s*\'\s*\/\s*579218131\s*\'\s*\/\s*(\d+)\s*\'\s*\/\s*(\d+)\s*\'$/
 
+// TODO: why are we copying this rather than using https://github.com/yogh333/starknetjs-signer-ledger/blob/main/src/ledger-signer.ts
```

Comment:
Good point! 
We implemented the signer interface before they got it upstream.
It diverges a little with what they have now. 
I'll test it out with the real device and their new c++ app and switch if ok.